game.py

```diff
@@ -487,8 +508,13 @@ def gameLoop():
 
             pygame.display.update()
             restart_game()
+            self.keys_pressed = {
+                pygame.K_LEFT: False,
+                pygame.K_RIGHT: False
+            }
             while 1:
-                for event in pygame.event.get():
+                self.py_events = pygame.event.get()
+                for event in self.py_events:
```

Comment:
Why this change?  Storing the events and reprocessing them?  Why not process them here?

Reply:
pygame.event.get() removes the events from the event queue ([documentation](https://www.pygame.org/docs/ref/event.html#pygame.event.get)). The key events are to be handled in check_inputs() function, whereas quit and pause check events were being handled in the gameLoop() itself.

Reply:
The event queue needs to be cleared so it doesn't get filled, are you saying some events aren't being handled?

Reply:
Yes, the pause event wasn't being handled correctly and the movements were janky when I tried to use pygame.event.get() in both _gameLoop_ and  _checkInputs_ functions. Storing the events in self.py_events solves this issue and also clears up the event queue.

Another alternative was to handle all the events in one place (in the gameLoop itself), but I chose to keep the checkInputs() function to keep the code more modular.

Reply:
Understandability trumps modularity.  This isn't how we do it everywhere else.  Please change the code so that the events are not _stored_ but are instead _processed_ within the loop that iterates over them, as before.

Reply:
[cfac48b](https://github.com/sugarlabs/ball-and-brick-activity/pull/24/commits/cfac48b4faf83e60c596f7fa06eab93d1d9399d4) resolves this.